examples/kuesa/car-scene/qml/main.qml

```diff
@@ -588,8 +588,69 @@ Item {
             PauseAnimation { duration: 1000 }
         }
 
```

Comment:
Would it make sense to move this into a DemoIdleAnimation element? I feel this might unnecessarily clutter the main.qml when people are getting started with this example.

Reply:
Agreed.

Reply:
done